logic/tokenCacheLogic.js

```diff
@@ -0,0 +1,53 @@
+const aeternity = require('../utils/aeternity.js');
+const cache = require('../utils/cache');
+const AsyncTipGeneratorsLogic = require('./asyncTipGeneratorsLogic');
+
+module.exports = class TokenCacheLogic {
+
+  constructor() {
+    this.init();
+  }
+
+  async init() {
+    await aeternity.init();
+    await cache.init(aeternity, () => {});
```

Comment:
do we need to init the cache twice?

Reply:
I haven't found a way not to do so, maybe you can have another look

Reply:
maybe we could add a global pre-start script that initialized the SDK and the cache here https://github.com/aeternity/tipping-community-backend/blob/master/server.js#L39